swift-blackjack-viewsTests/DeckSpec.swift

```diff
@@ -0,0 +1,48 @@
+//  DeckSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class DeckSpec: QuickSpec {
+    override func spec() {
+        describe("Deck") {
+            
+            var deck : Deck = Deck()
+            
+            beforeEach({ () -> () in
+                deck = Deck()
+            })
+            
+            describe("initializer") {
+                it("should set up the deck with 52 undealt cards") {
+                    expect(deck.description).to(contain("Cards Remaining: 52"))
```

Comment:
That seems like a really weird way to test this.. why not the count of the remaining cards array?